static/app/views/detectors/datasetConfig/utils/discoverSeries.tsx

```diff
@@ -79,9 +81,10 @@ export function getDiscoverSeriesQueryOptions({
         yAxis: aggregate,
         dataset,
         includePrevious: false,
-        partial: true,
```

Comment:
was this not doing anything?

Reply:
it was, but i think we don't want this in metric detectors